### ISSUE TITLE
Quantization aware training does not add weights quantization and fake quantization nodes to Conv2D layers without bias and followed by non-addition ops

### ISSUE THREAD

**Issue by protossw512** [tensorflow/tensorflow#31153]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: NA
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.14.0
- Python version: 3.6.6
- Bazel version (if compiling from source): NA
- GCC/Compiler version (if compiling from source): NA
- CUDA/cuDNN version: 10.1/7
- GPU model and memory: GTX 1080ti/12G

**Describe the current behavior**
Calling `tf.contrib.quantize.create_training_graph()` or `tf.contrib.quantize.create_eval_graph()` will not add weight quantization and activation quantization nodes to Conv2D layers with `use_bias=False, activation=None` and without following batch normalization layer.

**Describe the expected behavior**
A weights quantization node and an activation quantization node should be added to this layer, even it does not have biases, activation function, and batch normalization.

The issue seems to be related to [THIS](https://github.com/tensorflow/tensorflow/blob/dfe0d543aa6777b780f42f12fd503aebc71839d7/tensorflow/contrib/quantize/python/quantize.py#L529-L561) part

**Code to reproduce the issue**
```python
import tensorflow as tf
slim = tf.contrib.slim

if __name__ == '__main__':
    with tf.Session() as sess:
        input = tf.ones((2, 5, 5, 10))

        conv_bias = tf.keras.layers.Conv2D(
            15,
            kernel_size=3,
            dilation_rate=(1, 1),
            activation=None,
            use_bias=True,
            kernel_initializer=tf.truncated_normal_initializer(stddev=0.01),
            kernel_regularizer=slim.l2_regularizer(0.00005),
            kernel_constraint=tf.keras.constraints.UnitNorm(axis=2))

        conv_no_bias = tf.keras.layers.Conv2D(
            15,
            kernel_size=3,
            dilation_rate=(1, 1),
            activation=None,
            use_bias=False,
            kernel_initializer=tf.truncated_normal_initializer(stddev=0.01),
            kernel_regularizer=slim.l2_regularizer(0.00005),
            kernel_constraint=tf.keras.constraints.UnitNorm(axis=2))

        x_2 = conv_bias(input)
        x_1 = conv_no_bias(input)

        output = tf.concat((x_1, x_2), axis=3)
        tf.contrib.quantize.create_training_graph(quant_delay=0)
        writer = tf.summary.FileWriter('./bug_report', sess.graph)

```
**Other info / logs**
Graph from tensorboard:

![image](https://user-images.githubusercontent.com/18194209/62093263-810e9600-b22d-11e9-87d5-17703fa7542e.png)

**Comment by jvishnuvardhan** [comment]
I could reproduce the issue with TF1.14.0. Here is the [gist](https://colab.sandbox.google.com/gist/jvishnuvardhan/301d8c4a1c743e49639bd46161106fe9/tf31153.ipynb). Thanks!

**Comment by pianogGG** [comment]
I also encounter this problem, how do you solve this?@protossw512@jvishnuvardhan 
Thanks very much!

**Comment by jvishnuvardhan** [comment]
@pianogGG Can you please try `tf-nightly` and share a simple standalone code to reproduce the issue? Thanks!

**Comment by jvishnuvardhan** [comment]
@protossw512 Can you please try `tf-nightly` and share the updated code. I think there are no more updates to `contrib`. So it is better to try `TF2.x` or `tf-nightly`. Thanks!

**Comment by pianogGG** [comment]
> 
> 
> @pianogGG Can you please try `tf-nightly` and share a simple standalone code to reproduce the issue? Thanks!

Do you mean using tensorflow_model_optimization package to quantize ,right?

**Comment by pianogGG** [comment]
@jvishnuvardhan Do you mean using tensorflow_model_optimization package to quantize ,right?

**Comment by pianogGG** [comment]
@jvishnuvardhan  I find it can insert fakequant with conv without bias, did you modify something? If so, how modify?

**Comment by pianogGG** [comment]
Conv with relu6 but without bias  can insert fakequant node

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
